index.bs

```diff
@@ -669,11 +669,11 @@ Unlike WebGPU, this API does not intrinsically support custom shader authoring;
 
 The WebGPU API identifies <a href="https://gpuweb.github.io/gpuweb/#privacy-machine-artifacts">machine-specific artifacts</a> as a privacy consideration. Similarly, the WebNN API's compute unit scheduling may under certain circumstances introduce a fingerprint. However, similarly to WebGPU, such fingerprints are identical across most or all of the devices of each vendor, mitigating the concern. Furthermore, software implementations can be used to further eliminate such artifacts.
 
-The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. An {{MLDeviceType}} normatively indicates the kind of device and is one of: {{MLDeviceType/"cpu"}}, {{MLDeviceType/"gpu"}}, {{MLDeviceType/"npu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, thus this type can in some cases add two bits of entropy to the fingerprint. An {{MLPowerPreference}} indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
+The WebNN API defines developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. An {{MLPowerPreference}} indicates a preference as related to the desired low power consumption or high performance; this is considered a hint only and as such does not increase entropy of the fingerprint.
```

Comment:
The current text "select the most appropriate underlying execution device for the workload" sounds like a context can only support one device. But we also want to ["allow supporting multiple devices with one context."](https://github.com/webmachinelearning/webnn/blob/main/device-selection-explainer.md#proposed-minimum-viable-solution)?

How about just say "better select the underlying execution device for the workload"? That  aligns with the text "the user agent selects and creates the underlying execution device by taking into account these options" at line 732.

Reply:
Done.